### PR TITLE
chore: initial Release

### PR DESCRIPTION
**Pull request by shubham-stepsecurity**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: Bug report
```
Comment (varunsh-coder) [inline .github/ISSUE_TEMPLATE/bug-report.md:2]: We can remove this file

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+_Replace the bullet points below with your answers_
+
+### Description
```
Comment (varunsh-coder) [inline .github/pull_request_template.md:3]: We should also standardize on this

File: .github/codecov.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# see https://docs.codecov.com/docs/codecovyml-reference
+codecov:
```
Comment (varunsh-coder) [inline .github/codecov.yml:2]: We should standardize on this

File: .github/workflows/ci_cd.yml [hunk 4]
```diff
@@ -0,0 +1,590 @@
+name: CI/CD
+on:
+  # only on PRs into and merge to default branch
+  pull_request:
+    branches:
+      - master
```
Comment (varunsh-coder) [inline .github/workflows/ci_cd.yml:6]: We should be using main branch

File: .github/workflows/ci_cd.yml [hunk 5]
```diff
@@ -0,0 +1,590 @@
...
+        ci_integration_timeout_retry_on_timeout,
+        ci_integration_timeout_retry_on_error,
+        ci_windows,
+      ]
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "If this is hit, all tests successfully passed"
+
+  # runs on merge to default only
+  cd:
```
Comment (varunsh-coder) [inline .github/workflows/ci_cd.yml:567]: We should remove this release part

File: package.json [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nick-invision/retry.git"
+  },
+  "keywords": [],
+  "author": "Nick Fields",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/nick-invision/retry/issues"
```
Comment (varunsh-coder) [inline package.json:22]: this should point to step-security and we can remove author

**Review by h0x0er — APPROVED** [review]

**Pull request merged**